ui/src/components/system/DHCP.vue

```diff
@@ -92,13 +92,17 @@
         </td>
         <td class="fancy">{{ props.row.props.Description}}</td>
         <td class="fancy">
-          <span class="fa fa-desktop"></span>
+          <span class="fa fa-desktop margin"></span>
           {{props.row.props.IpAddress}}
         </td>
         <td class="fancy">
-          <span class="pficon pficon-plugged"></span>
+          <span class="pficon pficon-plugged margin"></span>
           {{props.row.props.MacAddress}}
         </td>
+        <td class="fancy">
+          <span v-show="props.row.props.LeaseExpiration" class="fa fa-hourglass-end margin"></span>
```

Comment:
remove `v-show="props.row.props.LeaseExpiration"`

Reply:
Why do you want to remove it? It will show the hourglasses even if there is not lease expiration.

Reply:
If there is no `lease expiration` a dash (`-`) is shown but the hourglass is always present to indicat the type of the field. But if you prefer to remove it no problem is only a cosmetis.

Reply:
Yes, ok. We can leave it shown. I proceed to update my PR.
Thanks.